### ISSUE TITLE
Where are the installation instructions?

### ISSUE THREAD

**Issue by uberhacker** [unicorn-fail/dreditor#275]
There are no manual installation instructions anywhere or did I miss them?  The only tidbit I have found is manually install the latest tag (version) from the Automated builds page.  Where do you install these files?

**Comment by markhalliwell** [comment]
https://github.com/unicorn-fail/dreditor/blob/1.x/CONTRIBUTING.md#manual-testing

**Comment by uberhacker** [comment]
Hey Mark,
Thanks for the quick response.  Sorry, I'm not following.  How do I install the dreditor.zip downloaded from https://dreditor.org/development/build in Chrome on Windows 10?  Where do I extract these files?

**Comment by markhalliwell** [comment]
You can extract them anywhere. Just select the extracted folder for step `#4` of https://github.com/unicorn-fail/dreditor/blob/1.x/CONTRIBUTING.md#chrome.

**Comment by uberhacker** [comment]
Hey Mark,
That worked but I needed to extract the files into a separate empty directory.  Extracting anywhere does not mean in a directory with other files which was the issue I had experienced. :)  Should there be a link at https://dreditor.org/install to https://github.com/unicorn-fail/dreditor/blob/1.x/CONTRIBUTING.md#manual-testing so as to not confuse other simpletons like myself?  Thanks for your quick reply.
